src/org/labkey/test/components/glassLibrary/ParentEntityEditPanel.java

```diff
@@ -353,14 +353,44 @@ public ParentEntityEditPanel removeType(String typeName)
         if(found)
         {
            elementCache().removeButton(index + 1).click();
-           getWrapper().waitFor(()-> getAllTypeCombo().size() < typeCombos.size(),
-                   "The type '" + typeName + "' was not successfully removed.",
-                   1_000);
+
+           // Need to check if this is removing the last/only type.
+           if(typeCombos.size() > 1)
+           {
+               // If it is not removing the last one can simple check that the count of combos is as expected.
+               getWrapper().waitFor(() -> getAllTypeCombo().size() < typeCombos.size(),
+                       "The type '" + typeName + "' was not successfully removed.",
+                       1_000);
+           }
+           else
+           {
+               // If this is removing the last/only one the count of combos will still be 1, so need a different check.
+               ReactSelect rs = getAllTypeCombo().get(0);
```

Comment:
This might be out of scope for this work, but if there is at all any way other than index to identify these controls, we ought to consider doing that.

Reply:
There might be, but I think that would be a challenge to address at a later time.